resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,248 @@
+
+    def word(self, length):
+        return ''.join(random.choice(string.lowercase) for i in range(length))
+
+    def set_name(self):
+        organism = random.choice(['Dictyostelium discoideum', 'Mus musculus', 'Homo sapiens'])
+        replicate = random.choice(['rep1', 'rep2', 'rep3', 'rep4', 'rep5'])
+        hour = random.choice(range(36))
+        kit = random.choice(['RiboZero', 'Nugen'])
+        group = random.choice(['treatment', 'control'])
+        if organism == 'Dictyostelium discoideum':
+            return 'Dd_{}_{}_hr_{}'.format(kit, replicate, hour)
+        if organism == 'Mus musculus':
+            return 'Mm_{}_{}_{}'.format(self.word(3), group, replicate)
+        if organism == 'Homo sapiens':
+            return 'Hs_{}_{}_{}'.format(self.word(3), group, replicate)
+
+    def gzopen(self, fname):
+        """Open Gzip files using io.BufferedReader."""
+        return io.TextIOWrapper(io.BufferedReader(gzip.open(fname)))
+
+    def escape_mongokey(self, key):
+        """Escape keys when serializing database entries"""
+        return key.replace('$', u'\uff04').replace('.', u'\uff0e').replace(' ', '_')
+
+    def generate_expressions(self, gene_ids, path):
+        """Generate random expression data"""
+        genes = {}
+        with gzip.open(os.path.join(path, 'expressions.tab.gz'), 'wb') as f:
+            f.write('{}\t{}\n'.format('Gene', 'Expression'))
+            with self.gzopen(gene_ids) as gene_ids:
+                all_genes = [line.strip() for line in gene_ids]
+                for g in all_genes:
+                    e = random.gammavariate(1, 100)
+                    f.write('{}\t{}\n'.format(g, e))
+                    genes[self.escape_mongokey(g)] = e
+
+        with open(os.path.join(path, 'expressions.json'), 'w') as j:
+            js_out = '{"exp_json":%s}' % json.dumps({'genes': genes}, separators=(',', ':'))
+            j.write(js_out)
+
+    def set_reads_descriptor(self):
+        barcodes = ['ATGCATGC', 'TACGTACG']
+        instruments = ['Illumina HiSeq X', 'Illumina HiSeq 3000', 'Illumina HiSeq 3000']
+        descriptor = {'barcode': random.choice(barcodes),
+                      'instrument_type': random.choice(instruments),
+                      'barcode_removed': True}
+        return descriptor
+
+    def set_sample_desciptor(self, name):
+        annotator = ['Billie Joe Armstrong', 'Dexter Holland', 'Mark Hoppus']
+        growth_protocol = "One vial of cells was thawed out and placed in a T75 \
+         flask in DMEM/F12 supplemented with 10% FBS and fed every 2 days. Cells \
+         were grown to ~50% confluence (~4-5 days), after which they were trypsinized."
+        library_prep = "Total RNAs were extracted using the RNeasy Mini Kit (Qiagen). \
+            RNA sequencing libraries were constructed under the standard protocol of \
+            illumina TruSeq RNA Prep Kit."
+        if name.startswith('Dd'):
```

Comment:
Maybe add comment for which case you are setting annotations. `Dd` doesn't mean anything to me :)